src/extensions/extra/brackets-fonts-viewer/main.js

```diff
@@ -43,6 +44,13 @@ define(function (require, exports, module) {
     /** @type {Array.<FontView>} */
     var _viewers = {};
 
+    // Get a name for this font for use in the CSS sample code
+    function _getFontName(file) {
+        var filename = file.name.substring(0, file.name.indexOf("."))
```

Comment:
Use the Filer Path module for this stuff vs. doing it by hand, probably you want `Path.basename(file.name)`